### PR TITLE
add FAQ and Background information pages

### PR DESCRIPTION
**Pull request by afeld**
[Preview](https://federalist-proxy.app.cloud.gov/preview/18f/boise/faq/)

We were getting these questions over and over...good to have a canonical place to point people. I've had @jameshupp tell me that FAQ pages are an anti-pattern, so tried to a) limit it to questions that we actually get frequently asked, and b) moved the content to a non-FAQ page, when I could think of a good place.

Also includes some minor formatting fixes.

### CODE REVIEW

File: _pages/faq.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+</details>
+
+<details markdown="1">
+  <summary>Are you going to "fix" ATOs?</summary>
+  No. See the [Knowns](../knowns/) page.
+</details>
+
+<details markdown="1">
+  <summary>What makes the team qualified to work on ATOs?</summary>
+  We aren't, really. At least not on our own. See the [Overview](../).
```
Comment (afeld) [inline _pages/faq.md:20]: Is this _too_ humble?

File: _pages/faq.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  **OAI**
+
+  * Has the convening power of the White House
+  * Isn't looking to "build"/"own" anything
+
+  **Project Boise**
+
+  * Is looking at ATOs from the perspective of "what can TTS offer?"
+  * Is talking to a broader range of stakeholders, including vendors
+</details>
```
Comment (afeld) [inline _pages/faq.md:66]: Probably makes sense to move this detail into the Areas of ATO Work page at some point.

File: _pages/faq.md (hunk 1 above)
Comment (andrewmaier) [reply _pages/faq.md:20]: Haha

**Review by andrewmaier — APPROVED** [review]
Made some slight changes. Looks good to me.

File: _pages/faq.md (hunk 2 above)
Comment (andrewmaier) [reply _pages/faq.md:66]: I agree with the idea, but I imagine both of these pages will change once we approach synthesizing, summarizing, and sharing out our discovery findings.

**Pull request merged**